docs/source/references/apollo-engine.md

```diff
@@ -0,0 +1,158 @@
+
+<h3 id="github-permissions">GitHub permissions</h3>
+
+GitHub’s OAuth service is used for read-only information about organizations and users. Engine does not need access rights to your source code or to any other sensitive data in its login system.
+
+If your Engine account is owned by a GitHub organization, then Engine will allow all members of that organization to access the account. As you add or remove team members from your Github org, Engine will know about that and accordingly update the authorization for those users.
+
+<h3 id="other-logins">Support for non-GitHub login</h3>
+
+Engine was built to mirror GitHub’s login and does not currently support other forms of login or team creation. Our current recommendation for teams who want to use Engine but do not use GitHub is:
+
+1. Create a free GitHub organization (empty, public).
+2. Add your desired team members to the organization.
+3. Hide membership of the organization if desired.
+4. Use this organization to collaborate on Engine.
+
+
+<h2 id="services">Services</h2>
+
+A service in Engine is an entity that is provisioned in the Engine cloud service that your server can report performance metrics and schema versions to via an API key. The information reported by your server is processed and stored in the Engine cloud service and made accessible to you through the Engine interface.
+
+<h3 id="creating-services">Creating a service</h3>
+
+Services in Engine have globally unique IDs, so we recommend that you prefix your service ID with the name of your company or organization. To create a service, you will need to select an account for that service to belong to. All members of the account will be able to see the service’s data and settings options.
+
+You can transfer services between any Engine account that you’re a member of. To transfer a service, visit its Settings page and change the “Endpoint owner” to whichever account you’d like.
+
+<h3 id="environments">Managing environments</h3>
+
+Services in Engine represent different projects/applications. Environments within your application can be tracked using [schema tags] (//TODO(dman): find link and figure out this recommendation) and managed within a single service.
+
+// TODO(dman): figure out if we want to add more schema tag information here
+
+#### API keys
+
+API keys can be added and removed form a service at any time. They are used to both send data to Engine (eg. server reporting configuration) and fetch information from Engine (eg. vs code extension configuration).
+
+You can manage your API keys on your service's settings page. It's recommended that you use one API key per function (eg. one key per data source) to have more granular control over how your Engine data is sent and accessed.
```

Comment:
I totally did not know you could connect multiple API keys! I feel like we should highlight this recommendation more prominently somewhere.

Reply:
Yep! I don't think this is what we want people doing long-term, but it's definitely what we recommend people do to manage their access right now.